jobqueue_features/clusters.py

```diff
@@ -513,14 +535,47 @@ def __init__(self, **kwargs):
                 logger.debug(warning)
             logger.debug("\n")
 
-    def _update_script_nodes(self):  # type: () -> None
+    def _update_script_nodes(self, **kwargs):  # type: () -> None
+        # If we're not in mpi_mode no need to do anything
         if not self.mpi_mode:
             return
+
         # When in MPI mode, after jobqueue has initialised we update the jobscript with
         # the `real` number of MPI tasks
         self.job_header = self.job_header.replace(
             "#SBATCH -n 1\n", "#SBATCH -n {}\n".format(self.mpi_tasks)
         )
 
+        # The default for jobqueue is not to use an MPI launcher (since it is not MPI
+        # aware). However, if self.fork_mpi=False then the tasks intended for this
+        # cluster are MPI-enabled. In order to give them an MPI environment we need to
+        # use our custom wrapper and launch with our MPI launcher
+        if not self.fork_mpi:
+            command_template = self._command_template
+            dask_worker_module = "distributed.cli.dask_worker"
+            if dask_worker_module in command_template:
+                command_template = command_template.replace(
+                    dask_worker_module, MPI_DASK_WRAPPER_MODULE
+                )
+            else:
+                raise RuntimeError(
+                    "Python module {} not found in command template:\n{}".format(
+                        dask_worker_module, command_template
+                    )
+                )
+            # The first part of the string is the python executable to use for the
+            # worker
+            [python, arguments] = command_template.split(" ", 1)
```

Comment:
You can actually omit list brackets since ince comma is for tuple unpacking from right side of `=` to the left side.
So it can looks like `python, arguments = command_template.split(' ', 1)`.